eras/shelley/impl/src/Cardano/Ledger/Shelley/TxBody.hs

```diff
@@ -431,7 +440,10 @@ deriving newtype instance CC.Crypto crypto => FromCBOR (TxId crypto)
 deriving newtype instance CC.Crypto crypto => NFData (TxId crypto)
 
 instance HeapWords (TxIn crypto) where
-  heapWords (TxInCompact txid ix) = 3 + HW.heapWordsUnpacked txid + HW.heapWordsUnpacked ix
+  heapWords (TxInCompact32 a _ _ _ ix) =
+    6 + (4 * HW.heapWordsUnpacked a) + HW.heapWordsUnpacked ix
```

Comment:
Are we sure that `heapWordsUnpacked (a :: Word64)` is zero? The answer here must be 6, so it must amount to 6 + 4 * 0 + 0. Can someone verify that the answer here does indeed come out to be simply 6?

Reply:
We have
```
instance HeapWords Word64 where
  heapWords _ = 2
```
and
```
heapWordsUnpacked :: HeapWords a => a -> Int
heapWordsUnpacked x = heapWords x - 2
```